File: projects/node-communication/node-subscriber.js
var zmq = require('zmq');
var subscriber = zmq.socket('sub');

var tcpport = 'tcp://macbook-cgarnier.local:9999';
var multicast = 'epgm://239.255.1.1:5555';

subscriber.connect(tcpport);
subscriber.subscribe('');

subscriber.on("message", function(reply) {
  console.log('Received message: ', reply.toString());
});

process.on('SIGINT', function() {
  subscriber.close();
  console.log('\nClosed');
});

File: projects/node-communication/node-subscriber.py
#!/usr/bin/python

import zmq

# Loading 0MQ context
context = zmq.Context()

# Configuration
tcpport = 'tcp://macbook-cgarnier.local:9999'
multicast = 'epgm://239.255.1.1:5555'

# Sockets
subscriber = context.socket(zmq.SUB)
subscriber.bind(tcpport)

while True:
	message = subscriber.recv_multipart()
	print message